source/NVDAObjects/UIA/edge.py

```diff
@@ -457,18 +457,19 @@ def _get_ariaProperties(self):
 	# RegEx to get the value for the aria-current property. This will be looking for a the value of 'current'
 	# in a list of strings like "something=true;current=date;". We want to capture one group, after the '='
 	# character and before the ';' character.
-	# This could be one of: True, "page", "step", "location", "date", "time"
-	RE_ARIA_CURRENT_PROP_VALUE = re.compile("current=(\w+);")
+	# This could be one of: "false", "true", "page", "step", "location", "date", "time"
+	# "false" is ignored, so it falls back to the default implementation, which is false

```

Comment:
Shouldn't the latter false be none?

Reply:
This is a comment about the regular expression, which is outside of the isCurrent method and the false here refers to the false as value of the aria-current attribute. That's also the reason why it is written without a capital F. So, I think this comment is correct.

Reply:
The comment on Line 461 confuses me a little too. It sounds like it is talking about the default implementation of the `isCurrent` method, perhaps you could clear this up? I think it would be clearer to say something like:
`# "false" is ignored by the regEx and will not produce a match`

Reply:
Thanks for the suggestion, I just commited the improved comment